proposed/phpdoc.md

```diff
@@ -1220,18 +1220,19 @@ The @method allows a class to know which 'magic' methods are callable.
 
 #### Syntax
 
-    @method [return type] [name]([type] [parameter], [...]) [description]
+    @method <static|> [name]([type] [parameter], [...])[: return type] [description]
```

Comment:
How about : `@method ["static"] <name>([<type>] [<parameter>] [, ...])[: <returnType>] [description]`

Optional literal static'.
Required name.
Optional type and parameter and repeatable.
Optional : return type.
Optional description.